tests/robustness/failpoint/network.go

```diff
@@ -193,21 +193,22 @@ func (f dropPeerNetworkFailpoint) Inject(ctx context.Context, t *testing.T, lg *
 	member := clus.Procs[rand.Int()%len(clus.Procs)]
 	proxy := member.PeerProxy()
 
-	proxy.ModifyRx(f.modifyPacket)
-	proxy.ModifyTx(f.modifyPacket)
+	if !f.shouldDropPacket() {
+		return nil, nil
+	}
+
+	proxy.BlackholeRx()
+	proxy.BlackholeTx()
```

Comment:
This is incorrect change, previously it allowed to drop random packets between two members, how it either drops all traffic or doesn't do anything. Looks like MotifyRx/ModifyTx was used after all. 

I think we should leave the L4 proxy afterall.

Reply:
Good catch @serathius! Maybe I am doing the cleanup a bit too aggressively :( 

I will roll back the changes!

Reply:
Rollback performed!